tests/unit/package_managers/test_general.py

```
@@ -166,26 +167,16 @@ def test_extract_git_info(url: str, nonstandard_info: Any) -> None:
 
-    class MockReadChunk:
-        def __init__(self) -> None:
-            """Create a call count."""
-            self.call_count = 0
-
-        async def read_chunk(self, size: int) -> bytes:
-            """Return a non-empty chunk for the first and second call, then an empty chunk."""
-            self.call_count += 1
-            chunks = {1: b"first_chunk-", 2: b"second_chunk-"}
-            return chunks.get(self.call_count, b"")
+    async def mock_iter_chunked(size: int) -> AsyncGenerator[bytes, None]:
```

Ruff (0.16.0)

[warning] Unused function argument: `size`

(ARG001)